blockonomics.php

```diff
@@ -154,6 +154,7 @@ public function installDB()
         Configuration::updateValue('BLOCKONOMICS_TIMEPERIOD', 10);
         Configuration::updateValue('BLOCKONOMICS_BTC', true);
         Configuration::updateValue('BLOCKONOMICS_BCH', false);
+        Configuration::updateValue('BLOCKONOMICS_LOGO_HEIGHT', "40");
```

Comment:
The default value should be set to 0 here

Reply:
Done